### ISSUE TITLE
Newly added XmlSerializerFormatAttribute tests failed in Helix run

### ISSUE THREAD

**Issue by imcarolwang** [dotnet/wcf#2356]
Error log:
```
Assert.Null() Failure
Expected: (null)
Actual:   IntParams { P1 = 5, P2 = 10 }
```

The tested Property is static in service hence it's shared by all clients, after test run once the property is assigned and not null any more which caused failure in next test case's pre-test validation stage.

**Comment by StephenBonikowsky** [comment]
@shmao Could you take a look at this?

**Comment by shmao** [comment]
This was fixed by #2358.
